platform/innovium/docker-syncd-invm-rpc/Dockerfile.j2

```diff
@@ -44,9 +48,9 @@ RUN wget https://github.com/nanomsg/nanomsg/archive/1.0.0.tar.gz \
  && cd ..               \
  && rm -fr nanomsg-1.0.0 \
```

Comment:
Can nanomsg instead be installed from the Debian repos, instead of being compiled here?

Reply:
Thanks, In the updated patch I am now installing through Debian repos